tests/foreman/cli/test_location.py

```diff
@@ -407,3 +408,29 @@ def test_negative_update_parent_with_child(self, request):
             Location.update({'id': location['id'], 'parent-id': location['id']})
         location = Location.info({'id': location['id']})
         assert location['parent'] == parent_location['name']
+
+    @pytest.mark.skip_if_open("BZ:1937009")
+    @pytest.mark.tier1
+    def test_positive_nested_location(self, request, module_org):
+        """View nested location in an organization
+
+        :id: cce4a33d-7743-47ef-b437-2539fdc73c44
+
+        :customerscenario: true
+
+        :BZ: 1937009
+
+        :expectedresults: Nested location can be viewed
+
+        :CaseImportance: Medium
+        """
+        parent_location = _location(request, {'organization-id': module_org.id})
+        location = _location(
+            request, {'parent-id': parent_location['id'], 'organization-id': module_org.id}
+        )
+        Org.add_location({'location-id': parent_location['id'], 'name': module_org.name})
+        Org.add_location({'location-id': location['id'], 'name': module_org.name})
+        loc_list = Location.list(
+            {'search': 'title=\\"%s\\"' % location['title'], 'organization-id': module_org.id}
```

Comment:
```suggestion
            {'search': f'title={location["title"]}', 'organization-id': module_org.id}
```
All the logic in the test looks good. I just think the f string is easier to read.

Reply:
Yes

> All the logic in the test looks good. I just think the f string is easier to read.

Yes. It would be easier to read in f string.  Same is updated.